src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag

```diff
@@ -11,18 +11,18 @@
     <div class="row">
       <div class="col-xs-12 col-md-6">
         <div class="row">
-          <div class="col-xs-12"
-              title="<%= Const.Tooltips.FEEDBACK_SESSION_SESSIONVISIBLELABEL %>"
-              data-toggle="tooltip"
-              data-placement="top">
+          <div class="col-xs-5">
```

Comment:
Any reason why you change `col-xs-12` to `col-xs-5` here and a few other places?